Redefinition of __rdtsc breaks modern GCC

Platform: Ubuntu 22.04 LTS
Compiler: gcc (Ubuntu 11.2.0-19ubuntu1) 11.2.0

gcc 11 now defines __rdtsc which results in a compile error.

```
/usr/lib/gcc/x86_64-linux-gnu/11/include/ia32intrin.h:112:1: error: redefinition of ‘long long unsigned int __rdtsc()’
  112 | __rdtsc (void)
      | ^~~~~~~
In file included from main_banded.cpp:39:
utils.h:60:34: note: ‘long long unsigned int __rdtsc()’ previously defined here
   60 | static inline unsigned long long __rdtsc(void)

```

--- a/benchmarks/bsw/utils.h
+++ b/benchmarks/bsw/utils.h
@@ -48,7 +48,7 @@
 
 #define xassert(cond, msg) if ((cond) == 0) _err_fatal_simple_core(__func__, msg)
 
-#if defined(__GNUC__) && !defined(__clang__)
+#if defined(__GNUC__) && __GNUC__ < 11 && !defined(__clang__)
 #if defined(__i386__)
 static inline unsigned long long __rdtsc(void)
 {

